### PR TITLE
Add MEASURE_BELT_TENSION macro and supporting shell script

### PR DESCRIPTION
**Pull request by blalor**
This helps visualize relative belt tension using the input shaper plugin.

### CODE REVIEW

**Comment by psyvision** [comment]
@blalor I have a version of this working, are you wishing to finish this MR or happy for me to add mine (using yours as insipiration)?

**Comment by blalor** [comment]
I have one last fix for the path to the script in the macro. I can finalize the PR today, or feel free to submit yours!

-- 
Brian Lalor (he/him)
***@***.***

**Comment by blalor** [comment]
After an embarrassing number of small issues (that's what I get for trying to shoot from the hip without actually using the repo locally), this is finally ready for review.

**Comment by guru-florida** [comment]
I just used this script and found it very useful. Worked perfectly and definitely goes hand-in-hand with the resonance shapers and fwiw I think it should be merged in pronto (it's 6 months pending btw). Thanks @blalor for such a great script!

**Comment by WolfHound01** [comment]
I've also use this code and it works flawlessly. 

+1 vote for merging ASAP.

**Comment by miklschmidt** [comment]
@guru-florida @WolfHound01 the only reason it's not merged yet is because the changes I requested are still pending.

**Comment by blalor** [comment]
Which changes are those @miklschmidt ? I don't see a review on this PR.

**Comment by miklschmidt** [comment]
> Which changes are those @miklschmidt ? I don't see a review on this PR.

Omg, that's the second time i forget to click the button.. My bad!

File: scripts/generate-belt-tension-graph.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+## but this has been reliable for me so far…
+sleep 10
+
+outdir=/home/pi/klipper_config/input_shaper
+if [ ! -d "${outdir}" ]; then
+    mkdir "${outdir}"
+fi
+
+~/klipper/scripts/graph_accelerometer.py \
+    -c /tmp/raw_data_axis*_belt-tension-*.csv \
```
[the remark this answers is not in the document]
Comment (blalor) [reply scripts/generate-belt-tension-graph.sh:15]: `TEST_RESONANCES … OUTPUT=raw_data NAME=belt-tension-upper` overwrites the files every time.  I don't recall the exact generated filename pattern, but it's something like `/tmp/raw_data_axis1,1_belt-tension-upper.csv` and `/tmp/raw_data_axis1,-1_belt-tension-lower.csv`.

[the remark this answers is not in the document]
Comment (miklschmidt) [reply scripts/generate-belt-tension-graph.sh:15]: Ah right, didn't see you had "NAME" on there, without it, it's suffixed with a timestamp. All good 👍

**Comment by miklschmidt** [comment]
@blalor Aight, I only have 1 last thing, could you rename the macro to "MEASURE_COREXY_BELT_TENSION"? Minion users can't use it for anything, and I think duplicating it across all corexy printers might be a little excessive. At least for now :D

**Comment by blalor** [comment]
Ok, ready for review with those changes, @miklschmidt!

**Comment by WolfHound01** [comment]
Nice! Thanks for the hard work to get this merge guys!

**Pull request merged**